plugins/s3-viewer-backend/src/service/router.ts

```diff
@@ -35,6 +35,10 @@ import {
   PolicyDecision,
 } from '@backstage/plugin-permission-common';
 
+/**
+ * @deprecated Use the new backend system instead
```

Comment:
Let's point people to https://github.com/spreadshirt/backstage-plugin-s3/tree/main/plugins/s3-viewer-backend for that.

Reply:
Done. Added the link in the 2 deprecation and also in the changeset